kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -178,6 +179,7 @@ private[kyuubi] class EngineRef(
 
     conf.set(HA_ZK_NAMESPACE, engineSpace)
     conf.set(HA_ZK_ENGINE_REF_ID, engineRefId)
+    conf.set(KYUUBI_ENGINE_SUBMIT_TIME_KEY, String.valueOf(System.currentTimeMillis()))
```

Comment:
can we pull out the start time of `ENGINE_INIT_TIMEOUT` ? we should keep `engine submit time` and `the timeout chekc of start time` consistent.

Reply:
done